.circleci/config.yml

```
@@ -67,6 +67,52 @@ jobs:
           fi
       - notify_main_failure
 
+  generate-website-mdx:
+    docker:
+      - image: *GOLANG_IMAGE
+    environment:
+      <<: *ENVIRONMENT
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: generate plugin docs
+          command: go run ./cmd/waypoint docs -website-mdx
+      - run:
+          name: generate cli docs
+          command: go run ./tools/gendocs
+      - run:
+          command: rm waypoint-restore.db.lock; git status
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+      - notify_main_failure
+
+  check-website-mdx:
+    docker:
+      - image: *EMBER_IMAGE
+    environment:
+      <<: *ENVIRONMENT
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          command: git status
+      - run:
+          command: cd ./website; npm install
+      - run:
+          command: cd ./website; npx --no-install next-hashicorp format
+      - run: |
+          if ! git diff --exit-code > /dev/null; then
+            echo "Git directory has website mdx changes. Run make gen/website-mdx"
+            echo "and validate that the corresponding website pages look acceptable."
```

What should someone do after this to resolve the failure? It's not clear to me reading this error message. Maybe another sentence to say what to do to resolve the issue would be good

```suggestion
            echo "'website' directory has unstaged mdx changes. Run make gen/website-mdx"
            echo "and validate that the corresponding website pages look acceptable."
```